src/clj/puppetlabs/puppetserver/certificate_authority.clj

```diff
@@ -1241,10 +1218,10 @@
   (if (fs/file? custom-oid-mapping-file)
     (let [oid-mappings (:oid_mapping (yaml/parse-string (slurp custom-oid-mapping-file)))]
       (into {} (for [[oid names] oid-mappings] [(name oid) (keyword (:shortname names))])))
-    (log/debugf "No custom OID mapping configuration file found at %s, custom OID mappings will not be loaded"
-                custom-oid-mapping-file)))
+    (log/debug (trs "No custom OID mapping configuration file found at {0}, custom OID mappings will not be loaded"
+                custom-oid-mapping-file))))
 
 (schema/defn ^:always-validate get-oid-mappings :- OIDMappings
   [custom-oid-mapping-file :- (schema/maybe schema/Str)]
   (merge (get-custom-oid-mappings custom-oid-mapping-file)
-         (clojure.set/map-invert puppet-short-names)))
+(clojure.set/map-invert puppet-short-names)))
```

Comment:
If you end up making any other changes (e.g. to bump to the new release of i18n) then it'd be nice to go ahead and put it back.  Not critical though.